spec/requests/cats_spec.rb

```diff
@@ -87,4 +87,20 @@
       expect(cats).to be_empty
     end
   end
+    describe 'cat validation error codes' do
+      it 'does not create a cat with a name' do
+        cat_params = {
+          cat:{
+          age: 3,
+          gender: 'Male',
+          breed: 'alley cat',
+          enjoys: 'Starting beef with everyone on his block'
+          }
+      }
+      post '/cats', params: cat_params
+      expect(response.status).to eq 422
+      cat = JSON.parse(response.body)
+      expect(cat['name']).to include "can't be blank"
+    end
+  end
```

Comment:
This is a good start to this spec file! Make similar tests for each of the validations: age, gender, breed, enjoys, and enjoys > 10 characters.